sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_v2/schema.yaml

```
@@ -0,0 +1,127 @@
+fields:
+- name: submission_date
+  type: DATE
+  mode: NULLABLE
+  description: Date of client activity
+- name: client_id
+  type: STRING
+  mode: NULLABLE
+  description: Unique ID for the client installation.
+- name: app_version
+  type: INTEGER
+  mode: NULLABLE
+- name: os
+  type: STRING
+  mode: NULLABLE
+- name: channel
+  type: STRING
+  mode: NULLABLE
+- name: locale
+  type: STRING
+  mode: NULLABLE
+- name: browser_version
+  type: STRING
+  mode: NULLABLE
+- name: country
+  type: STRING
+  mode: NULLABLE
+- name: newtab_homepage_category
+  type: STRING
+  mode: NULLABLE
+  description: Homepage category setting for Newtab
+- name: newtab_newtab_category
+  type: STRING
+  mode: NULLABLE
+  description: Newtab category setting
+- name: organic_content_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+  description: If organic stories are enabled
+- name: sponsored_content_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+  description: If sponsored stories are enabled
+- name: sponsored_topsites_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+  description: If sponsored topsites are enabled
+- name: organic_topsites_enabled
+  type: BOOLEAN
+  mode: NULLABLE
+  description: If organic topsites are enabled
```

Are these enabled by default and the correct description is instead `TRUE if the user has not disabled...`?
```suggestion
  description: TRUE if the user has enabled organic topsites in the browser.
```